doc/install_windows_manually.txt

```diff
@@ -8,6 +8,7 @@ Contents
 o Installation as a service
 o Execution as an application
 o Installation with a different service name
+o Initializing the Security Database
```

Comment:
May be better put 'Initializing the Security Database' as a separate document in doc? That will let me mention it in error message and forget this problem. Moreover that text applies not to windows-only (described method is OS independent).

Reply:
Split it out to "README.security_database.txt" to fit the style of the other files.